build-tools/packages/build-tools/package.json

```diff
@@ -96,5 +96,10 @@
 				"typescript"
 			]
 		}
+	},
+	"typeValidation": {
+		"disabled": false,
+		"broken": {},
+		"typeRollupFile": "dist/build-tools.alpha.d.ts"
```

Comment:
Can't we read this from the api extractor config?

Reply:
Or the exports field in the package.json.

Reply:
@tylerbutler For this build-tools package, can we leave it as the dist folder? This package doesn't have an api extractor config and the package.json doesn't have an exports field.

Reply:
What I mean is that the code should be able to determine the output path for the api-extractor rollups by loading the api-extractor config. For example, see what we do in setPackageTypesField:

https://github.com/microsoft/FluidFramework/blob/4e81f3d0c276d884ce3097982f80c90fb1096dee/build-tools/packages/build-cli/src/commands/release/setPackageTypesField.ts#L69-L71

We load the config using api-extractor then use that to get the paths of the output files. Hardcoding this will be brittle and annoying to maintain.

Reply:
If a package doesn't have an api-extractor config, then the code can just do what it does today. But if there is one, it should be loaded and consulted to determine the output paths of the rollups.

Reply:
@tylerbutler it still seems like packages should be able to specify local policy for this. I like the idea of checking the api-extractor config to get the name of the typings file (though we would also need to check the tsconfig file to correctly determine what the build output directory is), but how does it know _which_ typings file to use? This, at least, seems worth allowing packages to specify. By default we can use `@alpha`, but if a package doesn't generate alpha rollups, it can specify "beta" or "public" or something instead.

Thoughts?

Reply:
The api gives you the complete path to the output rollup file. You don't need to read tsconfig afaik.

The scenario you describe makes sense, but imo the setting should not be the path to the types file but the "rollup kind" that should be used for type testing.

Basically I don't want to have to update MORE places if/when we update the api-extractor config. Or stated differently, the path to the rollup should be written in exactly one place.

Reply:
That sounds good to me. The "rollup kind" would basically just be a key into the api-extractor config to get the matching file name.

Reply:
@WayneFerrao I think we still need to do this. Each package should specify (in its package.json's type-tests config) what kind of rollup it is comparing against. "Default" would presumably be `index.d.ts` (tsc's output). If a package specifies "alpha" (or one of the other tags), the tooling can check the api-extractor config to determine where the resulting d.ts file is for that kind.